crates/rolldown/tests/rolldown/topics/hmr/generate_patch_error/artifacts.snap

```diff
@@ -0,0 +1,301 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+---
+# Assets
+
+## main.js
+
+```js
+
+//#region hmr.js
+var hmr_exports = {};
+__export(hmr_exports, { foo: () => foo });
+const hmr_hot = __rolldown_runtime__.createModuleHotContext("hmr.js");
+__rolldown_runtime__.__toCommonJS(hmr_exports);
+__rolldown_runtime__.registerModule("hmr.js", { exports: hmr_exports });
+const foo = "hello";
+text$1(".hmr", foo);
+function text$1(el, text$2) {
+	document.querySelector(el).textContent = text$2;
+}
+hmr_hot.accept((mod) => {
+	if (mod) text$1(".hmr", mod.foo);
+});
+
+//#endregion
+//#region main.js
+var main_exports = {};
+const main_hot = __rolldown_runtime__.createModuleHotContext("main.js");
+__rolldown_runtime__.__toCommonJS(main_exports);
+__rolldown_runtime__.registerModule("main.js", { exports: main_exports });
+text(".app", "hello");
+function text(el, text$2) {
+	document.querySelector(el).textContent = text$2;
+}
+
+//#endregion
+```
+# HMR Step 0
+## Errors
+
+### PARSE_ERROR
+
+```text
+[PARSE_ERROR] Error: Unterminated string
+   ╭─[ hmr.js:1:20 ]
+   │
+ 1 │ export const foo = 'hello
+   │                    ───┬───  
+   │                       ╰───── 
+───╯
+
+```
+
+## Meta
+
+- full_reload: false
+- first_invalidated_by: None
+- is_self_accepting: false
+- full_reload_reason: None
+### Hmr Boundaries
+# HMR Step 1
+
+## Code
+
+```js
+var init_hmr_0 = __rolldown_runtime__.createEsmInitializer(function() {
+	try {
+		var ns_hmr = {};
+		__rolldown_runtime__.__export(ns_hmr, {
```

Comment:
Why the hmr code get included in the patch?

Reply:
I can, but it doesn't change anything, because I cannot generate the snapshot any more on my machine. Do you still want me to merge that PR?

Reply:
If so, nope. Let me merge it separately.